lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -507,6 +507,8 @@ getLocationInfoAsString(const SmallPtrSet<Operation *, 8> &ops,
     return str;
   case LoweringOptions::LocationInfoStyle::WrapInAtSquareBracket:
     return "@[" + str + ']';
+  case LoweringOptions::LocationInfoStyle::None:
+    return "";
```

Comment:
Could you return early without constructing `str`? i.e.
```cpp
if(style == LoweringOptions::LocationInfoStyle::None)
    return "";
auto str = getLocationInfoAsStringImpl(ops);
...
```

Reply:
done!